src/main.ts

```diff
@@ -87,6 +101,7 @@ async function run(): Promise<void> {
       addedModified = [] as string[]
     for (const file of files) {
       const filename = file.filename
+      core.debug(`${filename}`)
```

Comment:
you might make this message more descriptive, or just drop this line idk

Reply:
Indeed, I'll drop it, it's not needed, I just used it to debug the code logic